aten/src/ATen/xpu/XPUEvent.h

```diff
@@ -0,0 +1,145 @@
+#pragma once
+#include <ATen/xpu/XPUContext.h>
+#include <c10/util/Optional.h>
```

Comment:
We are in C++ 17 land, std optional is there for everyone to use
```suggestion
#include <optional>
```

Reply:
Done.